core/modules/ccontrol/testCControl.cc

```diff
@@ -486,7 +489,24 @@ static const std::vector< std::string > QUERIES = {
     "select shortName from Filter where shortName LIKE 'Z'",
 
     // DM-16532; this verifies that dotted IDs work for table+column ids. quoted dotted IDs are tested in the antlr4 test, below.
-    "SELECT Source.sourceId, Source.objectId From Source WHERE Source.objectId IN (386942193651348) ORDER BY Source.sourceId;"
+    "SELECT Source.sourceId, Source.objectId From Source WHERE Source.objectId IN (386942193651348) ORDER BY Source.sourceId;",
```

Comment:
that's a good suggestion. I think this comment got put on the wrong line though? Or there's something I don't understand; I think this would be applied in `BOOST_DATA_TEST_CASE(antlr4_compare, ANTLR4_COMPARE_QUERIES, queryInfo)` around line 961, right?

Reply:
This s just a general comment/suggestion, not pointing to any specific line of the unit test.